### ISSUE TITLE
model_train.py gets killed after 1 step, Please help

### ISSUE THREAD

**Issue by cyberclone12** [tensorflow/models#8593]
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
I0529 12:57:33.396360 139872418367296 session_manager.py:500] Running local_init_op.
INFO:tensorflow:Done running local_init_op.
I0529 12:57:33.526032 139872418367296 session_manager.py:502] Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into training/model.ckpt.
I0529 12:57:37.928450 139872418367296 basic_session_run_hooks.py:606] Saving checkpoints for 0 into training/model.ckpt.
INFO:tensorflow:loss = 4.3426676, step = 1
I0529 12:57:42.588260 139872418367296 basic_session_run_hooks.py:262] loss = 4.3426676, step = 1
Killed

**Comment by Piyush983** [comment]
In which TensorFlow you are performing CPU or GPU??

**Comment by ravikyram** [comment]
@cyberclone12 

It is difficult to understand your issues only based on the information you provided.Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
